org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/primitive/constraint/C_get_project_asts.java

```diff
@@ -0,0 +1,33 @@
+package org.metaborg.spoofax.core.stratego.primitive.constraint;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.Optional;
+
+import org.metaborg.spoofax.core.context.constraint.IConstraintContext;
+import org.spoofax.interpreter.core.InterpreterException;
+import org.spoofax.interpreter.terms.IStrategoString;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+import org.spoofax.interpreter.terms.IStrategoTuple;
+import org.spoofax.interpreter.terms.ITermFactory;
+
+public class C_get_project_asts extends ConstraintContextPrimitive {
```

Comment:
I would prefer to indicate in this name that it returns _analyzed_ ASTs, could it be renamed to `C_get_project_analyzed_asts`? Is also consistent with the name of the stratego wrapper strategy.

Reply:
Done